cw-orch-daemon/src/network_config.rs

```diff
@@ -0,0 +1,96 @@
+use std::collections::HashMap;
+
+use cw_orch_core::environment::ChainInfoOwned;
+
+use crate::env::default_state_folder;
+
+pub(crate) fn load(chain_id: &str) -> Option<ChainInfoOwned> {
```

Comment:
Do you prefer having this be a seperate function over having it be a fn in the `ChainInfoOwned` implementation?

Reply:
> Do you prefer having this be a seperate function over having it be a fn in the `ChainInfoOwned` implementation?

I would like to keep it inside cw-orch-daemon **only** what do you think?

Reply:
Yeah that's good.